ios/FluentUI/People Picker/ContactCollectionView.swift

```diff
@@ -0,0 +1,162 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+// MARK: ContactCollectionViewDelegate
+
+public typealias MSContactCollectionViewDelegate = ContactCollectionViewDelegate
```

Comment:
Use this instead:
@objc(MSContactCollectionViewDelegate)

side-note: the prefix is MSF, not just MS.